src/pages/resultsView/mutation/ResultsViewMutationTable.tsx

```diff
@@ -10,6 +10,7 @@ import ExonColumnFormatter from "shared/components/mutationTable/column/ExonColu
 
 export interface IResultsViewMutationTableProps extends IMutationTableProps {
     // add results view specific props here if needed
+    totalExon?:string;
```

Comment:
`totalExonNumber` or `totalNumberOfExons` would be better. 

Also, can this number be a `not a number`? If not, maybe declare it as a `number` instead of a `string`.

Reply:
@onursumer I think `totalNumberOfExons` should be number when it has exon data, but I'm not sure whether it always has data, so I put a `None` here as well. In this case maybe we could keep using string?

Reply:
let's keep it as a `string` for now